.circleci/config.yml

```diff
@@ -230,6 +230,46 @@ jobs:
       - run: sbt ci-release
       - save_deps_cache
 
+  publish-maven:
+    docker:
+      - image: circleci/openjdk:8
+    steps:
+      - checkout
+      - restore_deps_cache
+      - set-sdk-version
+      - run:
+          name: Publish maven plugin and archetype
+          command: |
+            cd maven-java
+            echo $PGP_SECRET | base64 -d | gpg --import --batch
+            # It seems ~/.m2 is not actually restored from the cache?
```

Comment:
We need to investigate it further. We may be doing something wrong. 

When I was bringing the maven code here, I was surprised to see that I have to call sbt publishM2 more then once in the same CI build. I was planning to call only once and reuse the local artifacts on the next jobs. Didn't work as expected.